### PR TITLE
Added function to initialize the Quaternions

### PR DESCRIPTION
**Pull request by jonasBoss**
initQuat() will calculate initial quaternion values based of magnetometer and accelerometer measurements.
This can be useful to reduce the initial error of the fusion algorithm.

### CODE REVIEW

File: src/SensorFusion.cpp [hunk 1]
```diff
@@ -431,6 +431,98 @@ void SF::MadgwickUpdate(float gx, float gy, float gz, float ax, float ay, float
...
+		//find index of Largest Euler parameter
+		int k=0;
+		for (int i=1; i<4; i++){
+			if (a[i] > a[k])
+				k = i;
+		}
+		//calculate that parameter
+		e[k] = sqrt(1 + 2 * a[k] - Trace)/2;
+		
+		Serial.print("k: ");
```
Comment (aster94) [inline src/SensorFusion.cpp]: cxan you comment out the print?

File: src/SensorFusion.cpp [hunk 2]
```diff
@@ -431,6 +431,98 @@ void SF::MadgwickUpdate(float gx, float gy, float gz, float ax, float ay, float
...
+		Serial.println(k);
```
Comment (aster94) [inline src/SensorFusion.cpp]: also this

**Comment by aster94** [comment]
nice code, but i would like to maintain the code clean without print statement

**Comment by jonasBoss** [comment]
I removed them, I missed those when cleaning up all my debugging statements.

**Comment by aster94** [comment]
Thanks for the PR, nice code

**Pull request merged**